spark/v2.4/spark/src/jmh/java/org/apache/iceberg/spark/source/WritersBenchmark.java

```diff
@@ -275,7 +275,7 @@ public void writePartitionedLegacyFanoutDataWriter(Blackhole blackhole) throws I
     TaskWriter<InternalRow> writer = new SparkPartitionedFanoutWriter(
         partitionedSpec, fileFormat(), appenders,
         fileFactory, io, TARGET_FILE_SIZE_IN_BYTES,
-        writeSchema, sparkWriteType);
+        writeSchema, sparkWriteType, table().properties());
```

Comment:
Can you add a default so that you don't need to change so many test and benchmark files?

Reply:
Sure